### PR TITLE
Adds affine and track_stats params to BatchNorm docstring

### PR DESCRIPTION
**Pull request by Mottl**
### PR Checklist

- [ ] Tests are added
- [ ] Entry in NEWS.md
- [*] Documentation, if applicable
- [ ] API changes require approval from a committer (different from the author, if applicable)

### CODE REVIEW

File: src/layers/normalise.jl [hunk 1]
```diff
@@ -199,6 +199,7 @@ end
 """
     BatchNorm(channels::Integer, λ=identity;
               initβ=zeros32, initγ=ones32,
+              affine=true, track_stats=true,
```
Comment (DhairyaLGandhi) [inline src/layers/normalise.jl:202]:
```suggestion
              affine = true, track_stats = true,
```

**Review by DhairyaLGandhi — APPROVED** [review]
Thanks! This lgtm, would you mind adding it for the other norm layers too?

**Comment by Mottl** [comment]
> Thanks! This lgtm, would you mind adding it for the other norm layers too?

Another layers already have this parameters in docstring

**Review by DhairyaLGandhi — APPROVED** [review]

**Comment by DhairyaLGandhi** [comment]
Ah, thanks!

bors r+

**Comment by DhairyaLGandhi** [comment]
Seems like our GPU CI queue is backed up.

**Comment by Mottl** [comment]
ping :-)

**Pull request merged**